ccpp/physics_namelists/input_GFS_v15_C96.nml

```diff
@@ -0,0 +1,104 @@
+&gfs_physics_nml
+       fhzero         = 6.
+       ldiag3d        = .true.
+       fhcyc          = 24.
+       nst_anl        = .true.
+       use_ufo        = .true.
+       pre_rad        = .false.
+       ncld           = 5
+       imp_physics    = 11
+       pdfcld         = .false.
+       fhswr          = 3600.
+       fhlwr          = 3600.
+       ialb           = 1
+       iems           = 1
+       IAER           = 111
+       ico2           = 2
+       isubc_sw       = 2
+       isubc_lw       = 2
+       isol           = 2
+       lwhtr          = .true.
+       swhtr          = .true.
+       cnvgwd         = .true.
+       shal_cnv       = .true.
+       cal_pre        = .false.
+       redrag         = .true.
+       dspheat        = .true.
+       hybedmf        = .true.
+       satmedmf       = .false.
+       shinhong       = .false.
+       do_ysu         = .false.
+       lheatstrg      = .false.
+       lgfdlmprad     = .false.
+       effr_in        = .false.
+       random_clds    = .false.
+       trans_trac     = .false.
+       cnvcld         = .true.
+       imfshalcnv     = 2
+       imfdeepcnv     = 2
+       cdmbgwd        = 0.125,3.0
+       prslrd0        = 0.
+       ivegsrc        = 1
+       isot           = 1
+       debug          = .false.
+       oz_phys        = .false.
+       oz_phys_2015   = .true.
+       h2o_phys       = .true.
+       nstf_name      = 2,1,1,0,5
+       xkzminv        = 0.3
+       xkzm_m         = 1.0
+       xkzm_h         = 1.0
+       do_sppt        = .false.
+       do_shum        = .false.
+       do_skeb        = .false.
+       do_sfcperts    = .false.
+/
+
+&gfdl_cloud_microphysics_nml
+       sedi_transport = .true.
+       do_sedi_heat = .false.
+       rad_snow = .true.
+       rad_graupel = .true.
+       rad_rain = .true.
+       const_vi = .F.
+       const_vs = .F.
+       const_vg = .F.
+       const_vr = .F.
+       vi_max = 1.
+       vs_max = 2.
+       vg_max = 12.
+       vr_max = 12.
+       qi_lim = 1.
+       prog_ccn = .false.
+       do_qa = .false.
+       fast_sat_adj = .false.
```

Comment:
FYI for future reference, fast_sat_adj was set to T in a previous commit. This must be set to F in the SCM because the fast saturation adjustment of the GFDL microphysics scheme is FV3-specific and cannot be run with the SCM. If it is set to T, the cloud fraction that is passed to radiation is not calculated.

Reply:
Understood.